content/docs/user-guide/basic-concepts/dependency.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Dependency
-match: [dependency, dependencies, depends]
+match: [dependency, dependencies, depends, input, inputs]
```

Comment:
Where are these tooltips used? E.g. `<abbr>input</abbr>`

Reply:
Same Q for **File Linking**, **Pipeline (DAG)**, and `run-caches`

Reply:
ah I see you're not a fan of future-proofing. Sure, I can revert the unused items.

Reply:
Thanks! Yeah we don't do it now so that was confusing. But TBH I don't have a strong opinion either way in this case 🙂

Reply:
removed